vendor/src/github.com/docker/engine-api/types/container/host_config.go

```diff
@@ -271,6 +272,22 @@ type UpdateConfig struct {
 	RestartPolicy RestartPolicy
 }
 
+// MountConfig holds attributes for configuring a container mount
+// This replaces the `Binds` field (kept for back-compat)
+// Not all fields are used in all cases, and some fields conflict based on `Type`
+type MountConfig struct {
+	Type        string // Type of mount, e.g. 'bind', 'volume'
+	Destination string // Path in the container to mount to
+	Source      string `json:",omitempty"` // Path on the host to mount from, only used for binds
+	Propagation string `json:",omitempty"` // [r]shared, r[private], only used for binds
+	Populate    bool   `json:",omitempty"` // Populate volume from container image path if the volume is empty
+	Writable    bool
+	// Sets selinux sharing context (binds only)
+	// Allows binds to either be allowed to be shared with other containers or private to the container
+	MCSAccessMode string                      `json:",omitempty"`
```

Comment:
Yes, but ideally we don't touch anything on the host FS.
The reason we have it on the command line is because it does currently create paths, and needs this in order to interact with selinux.

If we keep the PR as is and _don't_ auto-create host paths, then there would be no need to set the labels here.